exasol/ds/sandbox/runtime/ansible/general_setup_tasks.yml

```diff
@@ -3,6 +3,9 @@
     upgrade: yes
     update_cache: yes
   become: "{{need_sudo}}"
+- name: Install rsync
+  include_role:
+    name: rsync
```

Comment:
We could also skip installation of `rsync` for now and revert `tutorial.yml` to use `copy` again, as installation of `rsync` itself takes quite long and hence in total there is no significant benefit over `ansible.builtin.copy`.

Reply:
We probably will get more files in the long-term, so rsync might amortize better then copy